scripts/sv/callVariants.sh

```diff
@@ -21,12 +21,11 @@ echo "Assuming 2-bit reference: " "$TWOBIT_REFERENCE_LOCATION"
 
 cd "$GATK_DIR" 
 
-./gatk-launch CallVariantsFromAlignedContigsSpark \
-    --inputAlignments "$PROJECT_OUTPUT_DIR"/aligned_assemblies \
-    --inputAssemblies "$PROJECT_OUTPUT_DIR"/assembly_0 \
+./gatk-launch CallVariantsFromAlignedContigsSAMSpark \
```

Comment:
It was my impression that CallVariantsFromAlignedContigsSpark and CallVariantsFromAlignedContigsSAMSpark were identical except for input format.  It seemed to me that the SAM version was simpler and cleaner, because all of the necessary data is in one place in a single file.  Therefore, I changed FindBreakpoints to write a SAM file.  Am I mixed up?